platform-sdk/docs/proposals/template.md

```diff
@@ -0,0 +1,119 @@
+# Summary
+
+Give a 1-3 sentence summary of the capability in the design proposal.
+
+| Metadata           | Entities                            | 
+|--------------------|-------------------------------------|
+| Last Updated       | 2024-06-07  (Edward)                |
```

Comment:
I think we should try to deduplicate as much as possible. Last updated timestamp and committer is readily available in the commit history. I suggest we don't include it here.

Reply:
removed

Reply:
Could you push the commit with the removal please?

Reply:
@anthony-swirldslabs  Just pushed, sorry about that.